app/models/user.rb

```diff
@@ -872,6 +873,11 @@ def latest_micro_report_page
     [micro_reports.page.total_pages, 1].max
   end
 
+  def mark_mail_as_sent_before_auto_retire
+    self.sent_student_before_auto_retire_mail = true
+    save(validate: false)
```

Comment:
```suggestion
    update_column(sent_student_before_auto_retire_mail, true)
```

でいけるかなと思ったのですが、コールバックの実行や`updated_at`の値の変更をしたいなら今のままじゃないとダメですね。